packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -232,6 +233,12 @@ class VectorGraphicsCodec {
       buffer._putInt32(offsets.length);
       buffer._putFloat32List(offsets);
     }
+    if (transform != null) {
```

Comment:
let's also assert that transform != identity transform, and make sure callers pass null for identity.